docs/usecases/datastorage_for_institutions.rst

```diff
@@ -0,0 +1,314 @@
+   :alt: A simple, local version control workflow with datalad.
+   :figwidth: 80%
+
+   Trinity of research data handling: The data store (``$DATA``) is managed and
+   backed-up. The compute cluster (``$COMPUTE``) has an analysis-appropriate structure
+   with adequate resources, but just as users workstations/laptops (``$HOME``),
+   it is not concerned with data hosting.
+
+On their own machines (``$HOME``), researchers are free to do whatever they want
+as long as it is within the limits of their machines. The cluster (``$COMPUTE``)
+pulls the data exclusively from the data store (``$DATA``). Thus, within
+``$HOME``, researchers are free to explore data from ``$DATA`` as they wish,
+but scaling requires them to use ``$COMPUTE``. Results from ``$COMPUTE`` are pushed
+back to ``$DATA``, and hence anything that is relevant for a computation is tracked
+(and backed-up) there.
+
+The data store as a Git-annex RIA remote
+""""""""""""""""""""""""""""""""""""""""
+
+The distributed data store exists thanks to Git-annex: Any large file content in
+datasets is stored as a *value* in Git-annex's object tree. A *key*
+generated from its contents is checked into Git and used to reference the
+location of the value in the object tree [#f1]_. The object tree (or *keystore*)
+with the data contents can be located anywhere -- its location only needs to be
+encoded using a *special remote*. This configuration is done on an
+administrative, system-wide level, and users do not need to care or know
+about where data is stored, as they can access it just as easily as before.
+
+.. findoutmore:: What is a special remote?
+
+   A `special-remote <https://git-annex.branchable.com/special_remotes/>`_ is an
+   extension to Git's concept of remotes, and can enable Git-annex to transfer
+   data to and from places that are not Git repositories (e.g., cloud services
+   or external machines such as an HPC system). Don't envision a special-remote as a
+   physical place or location -- a special-remote is just a protocol that defines
+   the underlying *transport* of your files *to* and *from* a specific location.
+
+The machines in question, parts of an old compute cluster and parts of the
+supercomputer at the JSC, are configured to receive and store data using the
+Git-annex remote for indexed file archives (`RIA <https://libraries.io/pypi/ria-remote>`_)
+special remote. The Git-annex RIA-remote is similar to Git-annex's built-in
+`directory <https://git-annex.branchable.com/special_remotes/directory/>`_
+special remote, but distinct in certain aspects:
+
+- It allows read-access to (compressed) 7z archives, which is a useful
+  feature on systems where strong quotas on filesystem inodes might be imposed
+  on users, or where one wants to have compression gains.
+  This way, the entire key store (i.e., all data contents) of the
+  remote that serves as the data store can be put into an archive that uses
+  only a handful of inodes, while remaining fully accessible.
```

Comment:
My question as a sysadmin is: with how much overhead?

Latency for indexing is why we're using uncompressed 7z archives. They actually make it pretty darn usable.